Update FAQ with "Unused Build & Development Settings" (#1907)
vercel/docs: pages/docs/v2/platform/frequently-asked-questions.mdx

This ensures no requests are accidentally accepted.

## Unused Build & Development Settings

> A [Project](/docs/v2/platform/projects) has several settings that can be found in the dashboard. One section is called Build & Development Settings which is used to change the way a Project builds.

Reviewer: ```suggestion
A [Project](/docs/v2/platform/projects) has several settings that can be found in the dashboard. One of those sections is called "Build & Development Settings" and is used to change the way a Project is built.
```

> However, the Build & Development Settings are only applied to [zero-configuration](/guides/upgrade-to-zero-configuration) deployments.

Reviewer: ```suggestion
However, the Build & Development Settings are only applied to [zero-configuration](/guides/upgrade-to-zero-configuration) Deployments.
```

> When a deployment defines the [`builds`](/docs/configuration#project/builds) configuration property, the Build & Development Settings are ignored.

Reviewer: ```suggestion
If a Deployment defines the [`builds`](/docs/configuration#project/builds) configuration property, the Build & Development Settings are ignored.
```
